src/code/z_demo.c

```diff
@@ -1041,7 +1189,8 @@ void Cutscene_Command_Terminator(PlayState* play, CutsceneContext* csCtx, CsCmdB
                     }
                 }
                 break;
-            case 96:
+
+            case CS_DEST_DESERT_COLOSSUS_FROM_BLUE_WARP:
```

Comment:
Aren't those more "after getting x medallion" ?

As in, this isn't directly after the blue warp, there is the sage cutscene first, then this I assume

Reply:
this is tricky
the motivation of the name was because you literally come into the scene on a blue warp

I can understand why that would be confusing, so I was starting to change them. but there is a small issue:
`CS_DEST_SACRED_FOREST_MEADOW_FROM_BLUE_WARP`
as far as I can tell, this is unused. the entrance index it uses *is* the one where link comes into the meadow from a blue warp
but it goes unused because in the final game, you just get left in kokiri forest after the deku jr cutscene.
assuming the deku jr cutscene didnt exist, this *would be* the destination used from the chamber of sages.. but yeah, it wouldnt be right to call it "from chamber of sages" like im doing with the rest 

it can be called `CS_DEST_SACRED_FOREST_MEADOW_UNUSED` i guess, but yea

Reply:
went with `CS_DEST_SACRED_FOREST_MEADOW_WARP_PAD, // unused` since that is literally where you go
cant really say "from x" cause its not used, though we can pretty easily imply it *would* have been from chamber of sages